app/backend/src/index.ts

```diff
@@ -0,0 +1,44 @@
+import 'tsconfig-paths/register';
+import express, { Express, Request, Response } from "express";
+
+import dotenv from "dotenv";
+import morgan from 'morgan';
+import swaggerUi from 'swagger-ui-express';
+import swaggerJsdoc from "swagger-jsdoc";
+
+import registerRoutes from '@routes/index';
+
+dotenv.config();
+
+(async () => {
+  const app: Express = express();
+  const port = process.env.PORT || 3000;
+
+  //logger
+  app.use(morgan(':date[iso] :method :url :status :res[content-length] - :response-time ms'));
```

Comment:
config for dev and production. For dev we display log in console, for production we save log in .log file